shared/settings/landing/index.desktop.js

```diff
@@ -0,0 +1,289 @@
+          <Text onClick={() => onInfo()} type={'BodyPrimaryLink'} style={{marginRight: globalMargins.xtiny}}>
+            {level}
+          </Text>
+          <Text style={{...globalStyles.flex}} type={'BodySmall'}>
+            ({levelToPrice[level]})
+          </Text>
+        </Box>
+        {selected && <Meta title='Your Plan' style={{backgroundColor: globalColors.blue2}} />}
+      </Box>
+      <Box style={{...globalStyles.flexBoxRow, flex: 1}}>
+        <Text style={{...globalStyles.flex, marginRight: globalMargins.xtiny}} type={'BodySmall'}>
+          {levelToSpace[level]}
+        </Text>
+        <Stars level={level} />
+      </Box>
+      <Box style={{...globalStyles.flexBoxRow, flex: 1, justifyContent: 'flex-end'}}>
+        <PlanActionVariants variants={variants} />
+      </Box>
+    </Box>
+  )
+}
+
+function PaymentInfo ({name, last4Digits, isBroken, onChangePaymentInfo}: PaymentInfoType & {onChangePaymentInfo: () => void}) {
+  return (
+    <Box style={{...globalStyles.flexBoxColumn, marginTop: globalMargins.medium}}>
+      <Text
+        style={{color: globalColors.black_40}}
+        type='BodySmall'>
+        Your payment method
+      </Text>
+      <Divider />
+      <Box style={{...globalStyles.flexBoxRow, minHeight: ROW_HEIGHT, paddingLeft: globalMargins.xtiny, justifyContent: 'space-between', alignItems: 'center'}}>
+        <Box style={globalStyles.flexBoxColumn}>
+          <Text
+            type='Body'>
+            {name}
+          </Text>
+          <Text
+            style={{color: isBroken ? globalColors.red : globalColors.black_40}}
+            type='BodySmall'>
+            **** {last4Digits} {isBroken ? ' (broken)' : ''}
+          </Text>
+        </Box>
+        <Text type='BodySmall' link={true} style={{color: globalColors.blue}} onClick={onChangePaymentInfo}>
+          Update
+        </Text>
+      </Box>
+    </Box>
+
+  )
+}
+
+function Plan ({onInfo, onUpgrade, onDowngrade, freeSpace, freeSpacePercentage, selectedLevel, paymentInfo, onChangePaymentInfo, lowSpaceWarning}: PlanProps) {
+  return (
+    <Box style={globalStyles.flexBoxColumn}>
+      <Box style={globalStyles.flexBoxColumn}>
+        <Text
+          style={{color: globalColors.black_40, marginBottom: globalMargins.tiny}}
+          type='BodySmall'>
+          Your plan
+        </Text>
+        <Divider />
+      </Box>
+      {plans.map(p => (
+        <PlanLevelRow
+          key={p}
+          level={p}
+          onInfo={() => onInfo(p)}
+          variants={variantPropsHelper(selectedLevel, p, onDowngrade, onUpgrade, freeSpace, freeSpacePercentage, lowSpaceWarning)} />))}
+      {!!paymentInfo && <PaymentInfo {...paymentInfo} onChangePaymentInfo={onChangePaymentInfo} />}
+      {!!paymentInfo &&
+        <Text style={{marginTop: globalMargins.small}} type='BodySmall'>
+          * You only pay for data you write on Keybase. When you share a file, the recipient does not pay.
```

Comment:
There seems to be a weird character in between "file," and "the". Is it intentional?